cms/core/blocks/panels.py

```diff
@@ -3,27 +3,44 @@
 from wagtail import blocks
 
 
-class PanelBlock(blocks.StructBlock):
-    """DS Panel block.
-    https://service-manual.ons.gov.uk/design-system/components/panel
-    https://service-manual.ons.gov.uk/design-system/components/announcement-panel
-    https://service-manual.ons.gov.uk/design-system/components/success-panel
-    https://service-manual.ons.gov.uk/design-system/components/warning-panels.
-    """
-
-    variant = blocks.ChoiceBlock(
-        choices=[
-            ("warn", _("Warning")),
-            ("info", _("Information")),
-            ("announcement", "Announcement"),
-            ("error", "Error"),
-            ("success", "Success"),
-        ],
-        default="warn",
-    )
+class WarningPanelBlock(blocks.StructBlock):
     body = blocks.RichTextBlock(features=settings.RICH_TEXT_BASIC)
-    title = blocks.CharBlock(required=False, label=_("Title (optional)"))
+
+    def get_context(self, value: dict, parent_context: dict | None = None) -> dict:
+        context: dict = super().get_context(value, parent_context)
+        context["variant"] = "warn"
```

Comment:
If we remove the announcement panel, we also won't need to pass the variant to the context here - and the template could also be renamed for clarity.

Reply:
Will update this accordingly once I receive confirmation on if we still need the announcement panel.